app/res/layout/file_pref_dialog.xml

```diff
@@ -29,6 +29,12 @@
 
             <EditText
                 android:id="@android:id/edit"
+                android:layout_width="10dp"
```

Comment:
This is the editText whose value is saved in the preference. I've made it invisible and added another edittext which gets shown to the user which we keep using this one for saving preference value.